extensions.py

```diff
@@ -0,0 +1,39 @@
+import re
+import subprocess
+import unicodedata
+from datetime import date
+
+from jinja2.ext import Extension
+
+
+def git_user_name(default: str) -> str:
+    return subprocess.getoutput("git config user.name").strip() or default
+
+
+def git_user_email(default: str) -> str:
+    return subprocess.getoutput("git config user.email").strip() or default
+
+
+def slugify(value, separator="-"):
+    value = unicodedata.normalize("NFKD", str(value)).encode("ascii", "ignore").decode("ascii")
```

Comment:
Since we cast to a string here, I am assuming this function now accepts every object as an argument, right?
I'm however more into transferring the responsibility of providing a string to the function caller, i.e.

```suggestion
def slugify(value: str, separator: str = "-") -> str:
    value = unicodedata.normalize("NFKD", value).encode("ascii", "ignore").decode("ascii")
```

I'm wondering if NFKD isn't a slight overkill here, but if one wants their project name to be fancy shmancy then... why not I guess :)

Reply:
This function is copier from Django :slightly_smiling_face: But yeah, I don't expect anything else than strings to be passed to it anyway :+1: